src/node_modules/app/machines/builder/form.options.js

```diff
@@ -122,23 +126,13 @@ function computeLists (ctx) {
 function computeTypings (ctx) {
```

Comment:
`compute` doesn't seem to communicate what's going on, in this case we `set` (which is also easier and faster to read/write)